packages/syft/src/syft/core/tensor/smpc/mpc_tensor.py

```diff
@@ -56,6 +62,8 @@ def __init__(
             # ^This is unsecure and should be used with cautioness
             seed_shares = secrets.randbits(32)
 
+        self.seed_shares = seed_shares
+
         # TODO: We can get this from the the secret if the secret is local
```

Comment:
Should this todo be added to the backlog?

Reply:
added to https://app.clubhouse.io/openmined/story/1128/tech-debt-for-adp-smpc-demo?stories_sort_by=priority&stories_group_by=WORKFLOW_STATE